src/ModelContextProtocol/Protocol/Types/Resources/Resource.cs

```diff
@@ -3,7 +3,7 @@
 namespace ModelContextProtocol.Protocol.Types;
```

Comment:
Why did you move all of these files?

Reply:
Sorry, meant to revert that before pushing. I just wanted a way to find all the resources files in one quick view